tools/tojson.sh

```diff
@@ -28,8 +28,8 @@ then
 fi
 
 # Loop through all yaml files, convert to json
-rm -rf **/*.json
 cd src
+# rm -rf **/*.json
```

Comment:
Was this intentional?
I think we should either remove this line, or uncomment it.

Reply:
Removed. Thanks!